src/components/Quiz/Quiz.test.tsx

```
@@ -0,0 +1,1231 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
```

ESLint

[error] 'waitFor' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)